Sources/SnowManLogic/GameLogic/guessManager.swift

```diff
@@ -0,0 +1,64 @@
+public class GuessManager {
+	public var totalIncorrectGuessesAllowed: Int
+	public var correctGuesses: [String]
+	public var incorrectGuesses: [String]
+	public var guess: String
+
+	public init (totalIncorrectGuessesAllowed: Int) {
+		self.totalIncorrectGuessesAllowed = totalIncorrectGuessesAllowed
+		correctGuesses = []
+		incorrectGuesses = []
+		guess = ""
```

Comment:
I am confused about this because I feel like I do need an initial state of the guess. The game checks if `isGameOver`, which checks if `correctlyGuessedFullWord(word)`. That method does a string comparison, so if there is not a guess to compare (`guess == word`), I am going to get an error. Where would that guess come from if I have not yet taken a player turn? I could pass `false` if guess is null, but where would that null guess come from?